### PR TITLE
Update RAINS section in SCIONLab documentation

### PR DESCRIPTION
**Pull request by hyateh**
This change is [Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/188)

### CODE REVIEW

File: content/apps/rains.md [hunk 1]
```diff
@@ -6,14 +6,16 @@ nav_order: 70
...
 {% include alert type="Warning" content="The RAINS infrastructure in SCIONLab is currently NOT available" %}
 
 RAINS is an alternate protocol for Internet name resolution, designed as a replacement of the Domain Name System (DNS).
 The current implementation of RAINS can run on SCION, and serve SCION addresses.
 
-All of our apps can make use of RAINS to resolve hostnames, if a resolver is configured.
+~~All of our apps can make use of RAINS to resolve hostnames, if a resolver is configured.~~ (**The support is currently disabled and will be added back later.**)
 
-If you want to run your own RAINS servers, please refer to the documentation in the [READMEs](https://github.com/netsec-ethz/rains).
+~~If you want to run your own RAINS servers, please refer to the documentation in the [READMEs](https://github.com/netsec-ethz/rains).~~
```
Comment (hausheer) [inline content/apps/rains.md:18]: I suggest to update the URL to the README as well.

**Comment by hausheer** [comment]
Do we already have a test server running in SCIONLab that RAINS could be tested against? If so, it would be great to add it here. See also https://github.com/netsys-lab/scion-rains/issues/22

**Comment by matzf** [comment]
@hyateh Do you plan to update this following @hausheer's feedback? Let me know if you want me to merge this as is.

**Comment by hyateh** [comment]
> @hyateh Do you plan to update this following @hausheer's feedback? Let me know if you want me to merge this as is.

A student (Robin) is looking into this but is somehow blocked by not knowing which SCIONLab servers can be used to set up the test RAINS servers. Do you happen to have a clue? :-)

**Review by juagargi — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/188)* status: all files reviewed, 2 unresolved discussions / 0 of 1 LGTMs obtained / 0 of 1 approvals obtained

___
*[content/apps/rains.md, line 9 at r1](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/188#-MwavhyU3_DMbYV5Hotz:-MwavhyU3_DMbYV5Hou-:bhorbto) ([raw file](https://github.com/netsec-ethz/scion-tutorials/blob/abbe30389b8905d08867932f2b3e325b0fee514a/content/apps/rains.md#L9)):*
> ```Markdown
> # RAINS, Another Internet Naming Service
> 
> {% include alert type="Info" content="The development of RAINS is now maintained [here](https://github.com/netsys-lab/scion-rains)". To set up and test the latest RAINS manually, please refer to the [instructions](https://github.com/netsys-lab/scion-rains/tree/master/test/manual). %}
> ```

We still use the existing `netsec-ethz/rains` in `scion-apps`.
Is RAINS going to be maintained now in the proposed repository?

**Comment by hyateh** [comment]
> We still use the existing `netsec-ethz/rains` in `scion-apps`. Is RAINS going to be maintained now in the proposed repository?

Hi Juan, yes that's the current plan. The RAINS codebase will be hosted there by NetSys-Lab until at least the completion (expected July) of an externally funded project. After that, we need to figure out a proper transition of the codebase to RHINE.

**Comment by hausheer** [comment]
Superseded by https://github.com/netsec-ethz/scion-tutorials/pull/194

**Pull request closed without merging**